lib/osml/data_catalog/dc_dataplane.ts

```diff
@@ -214,11 +214,11 @@ export class DCDataplaneConfig extends BaseConfig {
       INGEST_CONTAINER_URI: "awsosml/osml-data-intake-ingest:latest",
       INGEST_CONTAINER_BUILD_TARGET: "ingest",
       INGEST_CONTAINER_REPOSITORY: "data-intake-ingest",
-      INGEST_CONTAINER_DOCKERFILE: "docker/Dockerfile.ingest",
+      INGEST_CONTAINER_DOCKERFILE: "docker/Dockerfile.ts.ingest",
```

Comment:
Why would this be `ts`? Since its completely different from DC and TS?

Reply:
Fixed-  refactor caught things it shouldn't have.